tests/src/JIT/Stress/ABI/Gen.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Numerics;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+
+namespace ABIStress
+{
+    internal static class Gen
```

Comment:
I think it is worth adding a comment, explaining what the high level overview of the class is doing. Either that or a small blurb on each member method.